rust/processor/migrations/2023-09-01-231247_events_v2_1/up.sql

```diff
@@ -0,0 +1,6 @@
+-- Your SQL goes here
+
+ALTER TABLE events DROP CONSTRAINT IF EXISTS fk_transaction_versions;
+
+ALTER TABLE events
+ALTER COLUMN "event_index" SET NOT NULL;
```

Comment:
where did this line come from?

Reply:
it was necessary to get the migrations to run as  CDB doesn't allow index on full fields